### ISSUE TITLE
Add keyring support to Storage

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#21]
```
Storage objects that store locally should support keyring, if not all samples 
then have at least one sample specifically that uses keyring.
```

Original issue reported on code.google.com by `jcgregorio@google.com` on 2 May 2011 at 3:23

**Comment by GoogleCodeExporter** [comment]
```
Just adding a note to consider using http://pypi.python.org/pypi/keyring
```

Original comment by `jcgregorio@google.com` on 7 Nov 2011 at 6:03

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision eb268fb96be9.
```

Original comment by `jcgregorio@google.com` on 1 Aug 2012 at 7:59
- Changed state: **Fixed**
